src/theme/components/text.tsx

```diff
@@ -17,102 +17,102 @@ type TextProps = Omit<TextPropsOriginal, 'css'>
 export const ThemedText = {
```

Comment:
a lot of these are now identical and we can reduce but I'm fine with that being a followup task as long as the tickets gets added

Reply:
agree, i went to do this but in the name of de-risking going to leave, will ping adrian to track this